### PR TITLE
Feature/word level progress tracking

### PR DESCRIPTION
**Pull request by Dhruv-1105**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] The changes have been tested successfully.

### CODE REVIEW

File: CHANGELOG.md
Comment (robingenz) [inline CHANGELOG.md:1]: Please revert the changes in this file.

File: package.json
Comment (robingenz) [inline package.json:1]: Please revert this change.

File: package-lock.json
Comment (robingenz) [inline package-lock.json:1]: Please revert this change.

**Review by robingenz — CHANGES_REQUESTED** [review]

**Comment by Dhruv-1105** [comment]
I have updated the commit as per changes requested. Please check and test it.

**Review by robingenz — APPROVED** [review]

**Comment by robingenz** [comment]
Please fix the workflow errors.

**Review by robingenz — APPROVED** [review]

**Pull request merged**